src/sdk.ts

```diff
@@ -135,14 +135,42 @@ s	 * @example
 	 *
 	 * @example
 	 * const payments = await client.getSubscriptionPayments();
+	 * const payments = await client.getSubscriptionPayments({ subscriptionID: 123 });
+	 * Legacy usage for backwards compatibility: pass planID as a number instead of options object
 	 * const payments = await client.getSubscriptionPayments(123);
 	 */
-	getSubscriptionPayments(planID?: number) {
+	getSubscriptionPayments(options?: number | {
+		planID?: number;
+		subscriptionID?: number;
+		isPaid?: boolean;
+		from?: Date;
+		to?: Date;
+		isOneOffCharge?: boolean;
+	}) {
+		const opts = (typeof options === 'number') ? { planID: options } : options;
+		const {
+			planID = null,
+			subscriptionID = null,
+			isPaid = null,
+			from = null,
+			to = null,
+			isOneOffCharge = null,
+		} = opts || {};
+
+		const body = {
```

Comment:
nit: just inline this body definition below in the `_request` call.

Reply:
Done in https://github.com/avaly/paddle-sdk/pull/82/commits/a99cac44504df95650f0a0b4332f88d9804fd482